### PR TITLE
docs: Added Windows Setup

### LINKED ISSUE #74: docs: Add Setup to run in Windows
## Description
As a USER for setting up this project locally in windows, I need the database and setup instructions so that I can easily run the project without facing any problem.

## Mocks
image

### Update [Required]
- [ ] Add all the necessary instructions that are required for setting up this project in windows.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by jeeva2812**
### Description

Added the process of setting up in Windows.

Fixes #74 

### Type of Change:

- Documentation

### Checklist:

- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,16 @@ Next follow these instructions.
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+    You may run the following commands for local setup of DB in Widows:
```
Comment (Amulya-coder) [inline README.md]:
```suggestion
    You may run the following commands for local setup of DB in Windows:
```
small typo mistake

File: README.md [hunk 2]
```diff
@@ -58,6 +68,12 @@ Next follow these instructions.
     source venv/bin/activate
     pip install -r requirements.txt
     ```
+    For Windows:
+    ```
+    virtualenv venv
+    venv\Scripts\activate
```
Comment (Amulya-coder) [inline README.md]:
```suggestion
    For Git Bash Users:
    ```
    source ./venv/Scripts/activate
     ``` 
   For Windows Command Line Users:   
    ``` venv\Scripts\activate```
```

@jeeva2812 mention about both the git bash and command-line users.

**Review by Amulya-coder — CHANGES_REQUESTED** [review]

**Comment by jeeva2812** [comment]
Done with the necessary changes.

File: README.md [hunk 3]
```diff
@@ -58,6 +68,17 @@ Next follow these instructions.
...
     pip install -r requirements.txt
     ```
+    For Windows:
+    ```
+    virtualenv venv
+    For Git Bash Users:
+    ```
+    source ./venv/Scripts/activate
+     ``` 
+   For Windows Command Line Users:
```
Comment (codesankalp) [inline README.md:78]: This line is not properly indented.

File: README.md [hunk 4]
```diff
@@ -58,6 +68,17 @@ Next follow these instructions.
     source venv/bin/activate
     pip install -r requirements.txt
     ```
+    For Windows:
+    ```
+    virtualenv venv
+    For Git Bash Users:
+    ```
+    source ./venv/Scripts/activate
```
Comment (codesankalp) [inline README.md]: This line must be in code blocks. 
And above line must not be in code blocks.
Please change it.

Comment (jeeva2812) [reply README.md]: I have done the required changes. Please do check.

Comment (codesankalp) [reply README.md]: `pip install -r requirements.txt` is repeating again and again. Can you separate it into another point?
Also, mention Linux in the first section of creating virtualenv.

Comment (jeeva2812) [reply README.md]: Made it less redundant now.

**Review by Amulya-coder — APPROVED** [review]
Everything looks fine to me 👍

**Review by codesankalp — APPROVED** [review]
Approving this as it looks fine.
Thanks for your contribution @jeeva2812 

@isabelcosta Needs your review here.

**Review by isabelcosta — APPROVED** [review]
Cool! Thank you for your contribution @jeeva2812 🎉

**Pull request merged**